src/main/include/TrajMath.h

```diff
@@ -32,12 +35,20 @@ namespace t34
             double limelight_angle
         );
 
+        void Periodic();
+
+        void PutTelemetry();
+
         double GetFiringAngleDeg();
 
         bool IsInRange();
 
         double GetDistanceFromTarget();
 
+        inline void InputMotorOutputPercent(const double percent) { m_motor_output = percent; }
+
+        inline void SetTargetHeightMeters(const double meters) { m_target_height_meters = meters; }
```

Comment:
Per our earlier conversation, what do you think about changing this to inches? We could even use the units library to enforce this:

```suggestion
        inline void SetTargetHeight(const inch_t height) { m_target_height = height; }
```

The above would, of course, require chaning the type of `m_target_height` to `units::length::inch_t`.

Reply:
This is something I can consider, but we have all of our math already made in meters. It seems like wasted time to spend it converting everything to be in imperial. Perhaps once we have everything made and we have extra time.